src/app/shared/dockstore.model.ts

```diff
@@ -103,5 +103,6 @@ export class Dockstore {
     enableMultiCloudLaunchWithDNAstack: false,
     enableOrcidExport: true,
     enableNotebooks: false,
+    enableMetrics: false,
```

Comment:
I thought we were enabling metrics by default? Not in the UI?

Reply:
Any thoughts @denis-yuen ? I was following https://ucsc-cgl.atlassian.net/browse/SEAB-5318 where it says to add a feature flag.

Reply:
> I thought we were enabling metrics by default? Not in the UI?

Hmmm, the API is always available. I was assuming that we would need more data (i.e. maybe we'd go live in a 1.14.1 or 1.14.2 when we get platform partners on board) before making it widely available in the UI

Reply:
Sounds good, I was just making sure. :)